jakarta-rest-4.0/7.0.0.Alpha1/jakarta-rest-tck-jdk-17.adoc

```diff
@@ -0,0 +1,57 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:resteasy_logo_100x.png[RESTeasy, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], the
```

Comment:
Another reference for requirements is the `How to Get Listed as a Compatible Product` section on https://jakarta.ee/compatibility/get-listed.  I do like your reference to https://www.eclipse.org/legal/tck.php as well.

Reply:
FYI https://jakarta.ee/committees/specification/tckprocess/ also describes certification requirements as well.

Reply:
Okay, should I update it?

Reply:
No, sorry to not be clear enough.  Just wanted to point out the process descriptions on the other two links.

Reply:
No worries, it's good to know!